

export const deletePharmacy = async()=>
{
    try{

    }
    catch(err)
    {
        
    }
}